.github/workflows/healthcheck-streaming.yml

```diff
@@ -60,11 +60,11 @@ jobs:
       uses: robinraju/release-downloader@v1
       with:
         repository: 'EpicGamesExt/PixelStreamingInfrastructure'
-        tag: 'minimal-streamer'
-        fileName: 'Minimal-PixelStreamer-5.5.7z'
+        tag: 'minimal-streamer-5.5'
+        fileName: 'Minimal-PixelStreamer-5.5-Win64-Development.7z'

```

Comment:
Upgraded to using our latest minimal streamer, as the other one was a bit out of date now.